src/python/ServerUtilities.py

```diff
@@ -198,7 +198,7 @@
                           ]}
 
 
-def NEW_USER_SANDBOX_EXCLUSIONS(tarmembers):  # TODO move to CRABClient !!!
+def NEW_USER_SANDBOX_EXCLUSIONS(tarmembers):  # This is only used in CRABClient !!! MOVE IT THERE
```

Comment:
Suggestion: Please refrain from changing code/comments that are unrelated to the PR.

Reply:
I know.. but I wanted go have a clean pylint report